tests/ekore/anomalous_dimensions/unpolarized/space_like/test_init.py

```diff
@@ -6,17 +6,18 @@
 from numpy.testing import assert_allclose, assert_almost_equal, assert_raises
 from scipy.linalg import expm
 
-from ekore import anomalous_dimensions as ad
+import ekore.anomalous_dimensions.unpolarized.space_like as ad_us
```

Comment:
Are you sure to run pre-commit? isort should group `ekore` together, I think

Reply:
Yes I am.